service/src/io/pedestal/http/impl/servlet_interceptor.clj

```diff
@@ -202,6 +202,7 @@
              ;                                                 :servlet-request servlet-request
              ;                                                 :servlet-response servlet-response})
              :async? servlet-async?)
+      (assoc-in [:request :context-path] (some-> servlet-request (.getContextPath)))
```

Comment:
I appreciate the defensive coding here and normally I'd require it, but this function is in the hot path for handling even simple requests.  If `servlet-request` is nil, I'd rather have the server blow up and return 500s.  No need to make this change, I'll make it in another branch/PR